changelogs/unreleased/gh-8196-fix-crash-box.internal.compact.md

```diff
@@ -0,0 +1,4 @@
+## bugfix/box
+
+* Fixed a crash when function box.internal.compact is running with id of
```

Comment:
This is an internal undocumented function. There's no point in mentioning a fix for misusing it in the changelog. Actually, I don't see much point in fixing this function at all... Why do you fix it?

Also, the ticket isn't in the plan and missing the target branch label. Please add.

Reply:
> Actually, I don't see much point in fixing this function at all...

To be clear, I wouldn't mind if we fixed this issue, but please address my other comments and drop the changelog entry.